### DISCUSSION TITLE
2023-09-06 Weekly Meeting 123 Notes/Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#415] [General]
Weekly Meeting 123 is happening on Wednesday, September 6, 2023 from 9:00 AM PDT (16:00 UTC) - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda: 

- shoutout to the fresh voices 
- champions moved to this Friday
- open collective did approve now, but I would like help (mm) 
- election? Where did we leave off with that planning? 

Continued: 

- working group check-ins
- Festival name vote ends today :checkered_flag: 

Last week: https://github.com/omigroup/omigroup/discussions/414

**Comment by mrmetaverse** [comment]
## Demos

**Comment by mrmetaverse** [comment]
## re: Election 
- we discussed the vacant chair seat and the potential for a new election. 
- First, we discussed the meta behind the election itself. Before nominating anyone, we are sharing thoughts about the idea of an election. 
- Shadow proposed focusing moreso on the tasks that anyone can do, rather than putting too much emphasis on a new set of chairs who "all the work may fall on". 
- if we don't have an election, what is our plan for a future election? and how do we support the existing elected leaders? 
- before nominating, we should consider what jobs need to be done, what the need is for a chair(s) and set a date or triggering event for a future election.

**Reply by indiebio** [reply]
Important for a chair to be consistent. 
I think a chair is also a "first port of call" for newcomers, like a mentor maybe

**Reply by indiebio** [reply]
Chairs are also about accountability. This links to governance, how do we gently manage accountability?

**Reply by mrmetaverse** [reply]
e.g. https://hackmd.io/@mrmetaverse/omi-tasks-audit
also: https://github.com/omigroup/omigroup/discussions/157
and https://github.com/omigroup/omigroup/discussions/165

**Comment by indiebio** [comment]
What are the big goals of OMI?
1. Implementers
2. Contributors

**Reply by indiebio** [reply]
A place that people can bring their solutions to. OMI has a review group, so protocols etc can be brought in, reviewed and supported as appropriate.

OMI is not the originators of ideas, because that becomes more difficult to advocate.

**Reply by mrmetaverse** [reply]
# The top two goals of OMI

- Implementors (# of) 
- Contributors (active members M/M; # of draft protocols submitted by outside sources)

**Reply by indiebio** [reply]
Why should people bring their projects here and not another place, e.g. MSF?
OMI may be more suited (while being collaborative) for open source people, independent... and provide networking for these, and also a networking effect through our community focused events (jimmy6dof comments)

"almost like a union for representing smaller developers at these larger forums"

**Reply by indiebio** [reply]
From shadow: 
I think we need to revisit doing stuff like this. 

Just showing who is affiliated, what they contributed, what their goals are, and how we all align.
![image](https://github.com/omigroup/omigroup/assets/23474368/846e570a-5cc7-4b07-977d-a852f7363beb)

**Reply by mrmetaverse** [reply]
"I care about both GOOD and OPEN protocols. If a protocol is open, thats great, but if its not very good, that doesn't drive us forward"

**Reply by indiebio** [reply]
We care about people using open protocols, not necessarily using our protocols, BUT they need to be good. So we need to, and are able to, control the quality of the protocols.

**Reply by mrmetaverse** [reply]
![image](https://github.com/omigroup/omigroup/assets/63426722/cb75abc3-e7e5-4a44-8e79-3b6821c7aac7)

**Reply by indiebio** [reply]
Encourage non-technical people to get more involved. Both in this group, and with creating/exploring the metaverse as well.

**Reply by mrmetaverse** [reply]
"An open protocol cannot be built by one person" 
"consensus and conversation are critical to a proper open protocol"

**Reply by indiebio** [reply]
Dogfood, show > tell:
Map the virtual worlds: which worlds are interoperable? (show results as we go on the website?).

On that, see KEYFRAME: I’m working on launching this digital museum 2024, reach out if you want to collab https://www.virtualworlds.museum/  a continuation of my mapping work.

**Reply by indiebio** [reply]
Lets find our first 5. 

5 examples of creators  using OMI_Seat, OMI_Physics, and OMI_Personality 

lets find our first 5 platforms to implement 1 OMI protocol. (maybe OMI SEAT and OMI Physics)

**Comment by indiebio** [comment]
We need a better public interface.
Discoverability, curation, aggregrators, public interface, like blogs, of what content / projects / etc are out there that we like

**Reply by indiebio** [reply]
Also, from a principles perspective, are we fully open, are we happy to incorporate more commercial companies too?

**Reply by indiebio** [reply]
Everyone wants something different, but we can take aim together
from M3 charter https://mirror.xyz/m3org.eth/A7LmzSoB36dF8-rLkT8onBO_7vjAzEU1b4LxCeHxCl4

**Comment by indiebio** [comment]
Lets find our first 5. 

5 examples of creators  using OMI_Seat, OMI_Physics, and OMI_Personality 

lets find our first 5 platforms to implement 1 OMI protocol. (maybe OMI SEAT and OMI Physics)

OMI_seat and OMI_spawn_point are two low-hanging fruits, they are simple, easy to implement, and are unlikely to change in the future. We could spearhead with getting these used by 5 platforms.

**Reply by mrmetaverse** [reply]
Thanks @indiebio! I have created this ticket for review: https://github.com/omigroup/omigroup/issues/416

**Reply by indiebio** [reply]
if interested in logistics for virtual event production: https://github.com/M3-org/logistics
